cypress/support/service/administration/fixture.service.js

```diff
@@ -47,5 +47,3 @@ class AdminFixtureService {
 }
 
 module.exports = AdminFixtureService;
-
-global.AdminFixtureService = new AdminFixtureService();
```

Comment:
This prevented  me from debugging the services, because the runtime was not yet booted.  I wanted to use commands in the constructor. Do these serve any purpose?

Reply:
Good question, I don't remember anymore 🙈  @klarstil do you?

Reply:
This isn't possible here. What we do here is binding the instanced service class to the `global` object of Node.js (see: https://reactgo.com/node-global-variable/) to have them available anywhere within tests and the commands without importing and instantiating them individually.

Reply:
Seems to work by chance without requiring the fixture services. Could these not be required by the index.js?

Reply:
I've reverted these changes to the services